libs/common/src/lib/ai-models/ai-models.component.ts

```diff
@@ -88,17 +84,11 @@ export class AiModelsComponent implements OnInit {
         tap(({ kbConfig, learningSchema }) => {
           this.kbConfigBackup = kbConfig;
           this.learningConfigurations = learningSchema;
```

Comment:
knowing there’s nothing left in the pipe after this step, it could be done directly in the subscribe method.